Makefile

```diff
@@ -139,12 +141,12 @@ validate:
 auto-review: vendor
 	vendor/bin/phpunit --group=auto-review
 
-build/bin/infection.phar: bin src vendor box.phar box.json
+build/bin/infection.phar: $(shell find bin/ src/ -type f) box.phar box.json .git/HEAD
 	php box.phar compile
 
 box.json: box.json.dist
 	cat box.json.dist | sed -E 's/\"key\": \".+\",//g' | sed -E 's/\"algorithm\": \".+\",//g' > box.json
 
 box.phar:
-	wget https://github.com/humbug/box/releases/download/3.0.0-alpha.5/box.phar
+	wget $(BOX_URL)
```

Comment:
What is the reason that we use a variable here, but not for the other phars?

Reply:
That's a good point. We should.

Reply:
OK, done in 343c410